File: ERPServer/Shared/Shared/Entities/Setup/ProductTrackingDTO.cs
﻿using Data.Entities.Shared;
using Shared.Entities.Shared;
using Shared.Enums;
using System;


namespace Shared.Entities.Setup
{
    public class ProductTrackingDTO : BaseDTO
    {
        public long ProductId { get; set; }
        public string Date { get; set; }
        public string OldData { get; set; }
        public string NewData { get; set; }
        public string CreatedByUsername { get; set; }
        public ProductProcessTypeEnum ProductProcessTypeId { get; set; }
    }
}

File: erp-app/src/app/modules/setup/models/product-tracking.dto.ts
import { ProductProcessTypeEnum } from "src/app/shared/enums/product-process-type.enum";

export class ProductTrackingDTO {
    productId: number;
    date: string;
    oldData: string;
    newData: string;
    createdByUsername: string;
    productProcessTypeId: ProductProcessTypeEnum;
}